client/src/components/Form/components/formComponents/ExpandableField.tsx

```diff
@@ -1,14 +1,24 @@
-import './ExpandableField.less';
-
-import React, { ReactChild, ReactChildren, ReactNode } from 'react';
+import React, { ReactChild, ReactChildren, ReactNode, useEffect, useRef } from 'react';
 
 interface ExpandableFieldProps {
     show?: boolean;
     children: ReactNode | ReactChild | ReactChildren;
 }
 
 const ExpandableField = ({ show = false, children }: ExpandableFieldProps) => {
-    return <div className={`expandable-field ${show ? 'expandable-field--expanded' : ''}`}>{children}</div>;
+    const contentRef = useRef<HTMLDivElement>(null);
+
+    useEffect(() => {
+        if (show) {
+            contentRef.current?.focus();
```

Comment:
tanken med denne er at en skjermleser skal forstå at det dukker opp et nytt element. Har det noe for seg?

Reply:
Usikker på om denne faktisk gjør noe. Fokuset ender seg ikke til Content så vidt jeg kan se (via bruk av tastatur for interaksjon)